objects/activities.json

```diff
@@ -0,0 +1,50 @@
+{
+    "description": "Activities related to the current finding",
```

Comment:
Is it correct to refer to these things as the _activities_ related to the finding? I think that the more generic term _events_ might be more appropriate and flexible. What I'm getting at is that all activities are events but not all events are activities. I realise that right now, all of the attributes you've shown below are activities. But it is conceieveable that there could be non-activity events associated with a finding in the future. For example, a `Vulnerability Finding` or `Compliance Finding` might contribute towards a `Detection Finding`.